### PR TITLE
Update all dependencies

### PR DESCRIPTION
**Pull request by jkeiser**

### CODE REVIEW

File: kitchen-tests/Berksfile.lock [hunk 1]
```diff
@@ -3,7 +3,7 @@ DEPENDENCIES
     path: cookbooks/base
   php (~> 1.5.0)
   resolver
-    git: git://github.com/chef-cookbooks/resolver.git
+    git: https://github.com/chef-cookbooks/resolver.git
```
Comment (afiune) [inline kitchen-tests/Berksfile.lock]: change from `git://` to `https://` is desired?

**Comment by mwrock** [comment]
:+1:

File: omnibus_overrides.rb [hunk 2]
```diff
@@ -10,10 +10,10 @@
...
 override "makedepend", version: "1.0.5"
 override "ncurses", version: "5.9"
-override "openssl", version: "1.0.2h"
 override "pkg-config-lite", version: "0.28-1"
 override "ruby", version: "2.3.1"
 override "ruby-windows-devkit-bash", version: "3.1.23-4-msys-1.0.18"
 override "util-macros", version: "1.19.0"
 override "xproto", version: "7.0.28"
 override "zlib", version: "1.2.8"
+override "openssl", version: "1.0.2h"
```
Comment (afiune) [inline omnibus_overrides.rb]: Out of curiosity, you changed the order of the override for any particular reason? 🤔

File: kitchen-tests/Berksfile.lock (hunk 1 above)
Comment (jkeiser) [inline kitchen-tests/Berksfile.lock]: Yes, https is accepted by cookbook security stuff and git is not.

Comment (mwrock) [inline kitchen-tests/Berksfile.lock]: yeah I think we always prefer https. Github started timing out on jenkins a few weeks back and switching to https fixed that

File: omnibus_overrides.rb (hunk 2 above)
Comment (jkeiser) [inline omnibus_overrides.rb]: This is done automatically by `rake dependencies`

**Comment by afiune** [comment]
Left two small comments that will not stop this change to be merged. Code LGTM! 👍

**Comment by btm** [comment]
:+1:

**Pull request merged**